### PR TITLE
Mj add post contrib reminder feedback button

### PR DESCRIPTION
**Pull request by michaelbjacobson**
Adds a feedback request to the post contribution reminder copy, as per [this card](https://trello.com/c/85KWaulR).

The button opens a new tab/page and links directly to Survey Monkey.

Please see screenshots below for styling implementation.

**Laptop**
laptop
<br>

**Tablet**
tablet
<br>

**Smartphone**
smartphone
<br>

### CODE REVIEW

**Comment by hyunaleecasual** [comment]
Hi @michaelbjacobson, Thank you so much for doing this! I believe @ionamckendrick would comment later, but I was looking at the design and think the button should be smaller? What do you think? It's simply because asking this survey is an additional step, I don't think we want to push it as strong as '44px size button' but 36px size button. Does it make sense? Ha ha 
Screenshot 2020-05-20 at 14 42 50

**Review by ionamckendrick — APPROVED** [review]
As long as this isn't you editing the Epic down to two lines 🤣 ... it looks great! 👏

**Comment by ionamckendrick** [comment]
> Hi @michaelbjacobson, Thank you so much for doing this! I believe @ionamckendrick would comment later, but I was looking at the design and think the button should be smaller? What do you think? It's simply because asking this survey is an additional step, I don't think we want to push it as strong as '44px size button' but 36px size button. Does it make sense? Ha ha
> Screenshot 2020-05-20 at 14 42 50

this is correct! sorry, I missed this. Thanks @hyunaleecasual 
Would you mind making it smaller @michaelbjacobson ?

**Comment by michaelbjacobson** [comment]
No problem at all @hyunaleecasual @ionamckendrick! Sorry I didn't pick up on that from the Figma wireframes! 

Please see the screenshot below for the new styling.

tablet (new)

**Review by hyunaleecasual — APPROVED** [review]
Awh this is great!! Thank you so much!

**Review by paulbrown1982 — APPROVED** [review]
👍 LGTM

**Comment by michaelbjacobson** [comment]
Merging into master and deleting remote branch.

File: static/src/javascripts/projects/common/modules/commercial/epic-reminder-email-signup.js [hunk 1]
```diff
@@ -13,14 +13,15 @@ type Fields = {
...
+    surveyRequest: HTMLElement,
     closeButton: HTMLElement,
     reminderPrompt: HTMLElement,
     reminderToggle: HTMLInputElement,
 };
 
 const isValidEmail = (email: string) => {
     const re = /[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?/;
-    return re.test(email);
+    return (email.replace(/\s+/g, '') === email) ? re.test(email) : false
```
Comment (tomrf1) [inline static/src/javascripts/projects/common/modules/commercial/epic-reminder-email-signup.js:24]: is this necessary now that you have the `trim` before?

**Review by tomrf1 — COMMENTED** [review]
great!

**Pull request merged**